packages/react-magma-dom/src/components/InputBase/index.tsx

```diff
@@ -114,6 +114,10 @@ export interface InputBaseProps
    * Boolean for whether this is a Password Input or not
    */
   isPasswordInput?: boolean;
+  /**
+   * String to determine width of input
+   */
+  width: string;
```

Comment:
I think we want to make this prop optional:
```suggestion
  width?: string;
```